packages/graphql-yoga/src/index.ts

```diff
@@ -0,0 +1,116 @@
+import fastify, { FastifyInstance } from 'fastify'
+import pino from 'pino'
+import { renderGraphiQL, sendResult, shouldRenderGraphiQL } from 'graphql-helix'
+import {
+  BaseNodeGraphQLServer,
+  BaseNodeGraphQLServerOptions,
+} from '@graphql-yoga/core'
+import { EnvelopError as GraphQLServerError } from '@envelop/core'
+import { getHttpRequest } from './request'
+
+/**
+ * Configuration options for the server
+ */
+export type GraphQLServerOptions = BaseNodeGraphQLServerOptions & {}
+
+/**
+ * Create a simple yet powerful GraphQL server ready for production workloads.
+ * Spec compliant server that supports bleeding edge GraphQL features without any vendor lock-ins.
+ *
+ * Comes baked in with:
+ *
+ * - Fastify - Fast an low overhead framework for Node.js
+ * - Envelop - Plugin system for GraphQL
+ * - GraphQL Helix - Extensible and Framework agnostic GraphQL server
+ * - GraphiQL - GraphQL IDE for your browser
+ * - Pino - Super fast, low overhead Node.js logger
+ *
+ * Example:
+ * ```ts
+ *  import { schema } from './schema'
+ *   // Provide a GraphQL schema
+ *  const server = new GraphQLServer({ schema })
+ *  // Start the server. Defaults to http://localhost:4000/graphql
+ *  server.start()
+ * ```
+ */
+export class GraphQLServer extends BaseNodeGraphQLServer {
+  private _server: FastifyInstance
+  private _getHttpRequest = getHttpRequest
```

Comment:
Thoughts on if I move this to `BaseNodeGraphQLServer` and have an overloaded `getHttpRequest` handler that supports `http`,`express`, `Fastify`?

Reply:
Do we need to have this done if we'll use W3C Request?

Reply:
ohh right then we don't need it here